File: Top 100 Codes/05-Operations on Strings/05-Toggle each character in a string/Toggleeachcharacterinastring.cpp
#include<iostream>
#include<string>
using namespace std;

string toggleString(string str){
    for(int i = 0; i < str.length(); i++){
        if(str[i] >= 'a' && str[i] <= 'z')
        {
            str[i] = str[i] - 32; // Convert lowerCase to UpperCase
        }
        else if(str[i] >= 'A' && str[i] <= 'Z')
        {
            str[i] = str[i] + 32; // Convert UpperCase to LowerCase
        }
    }
    return str;
}
int main(){
    string str;
    cout<<"Enter a String: ";
    getline(cin, str);

    cout<<"Toggled String: "<<toggleString(str)<<endl;
    return 0;
}

File: Top 100 Codes/05-Operations on Strings/05-Toggle each character in a string/Toggleeachcharacterinastring.py
def toggle_string(s):
    toggled = ""
    for i in range(len(s)):
        if 'a' <= s[i] <= 'z':
            toggled += chr(ord(s[i]) - 32) # Lowercase to uppercase
        elif 'A' <= s[i] <= 'Z':
            toggled += chr(ord(s[i]) + 32) # Uppercase to lowercase
        else:
            toggled += s[i] # Special characters
    return toggled

input_str = input("Enter a String: ")
print("Toggled string: ", toggle_string(input_str))